app/workers/mails.php

```diff
@@ -32,67 +29,12 @@ public function run(): void
             return;
         }
 
-        $project = new Document($this->args['project'] ?? []);
-        $user = new Document($this->args['user'] ?? []);
-        $team = new Document($this->args['team'] ?? []);
-        $payload = $this->args['payload'] ?? [];
 
         $recipient = $this->args['recipient'];
-        $url = $this->args['url'];
+        $subject = $this->args['subject'];
         $name = $this->args['name'];
-        $type = $this->args['type'];
-        $prefix = $this->getPrefix($type);
-        $locale = new Locale($this->args['locale']);
-        $projectName = $project->isEmpty() ? 'Console' : $project->getAttribute('name', '[APP-NAME]');
-
-        if (!$this->doesLocaleExist($locale, $prefix)) {
-            $locale->setDefault('en');
-        }
-
-        $from = $project->isEmpty() || $project->getId() === 'console' ? '' : \sprintf($locale->getText('emails.sender'), $projectName);
-        $body = Template::fromFile(__DIR__ . '/../config/locale/templates/email-base.tpl');
-        $subject = '';
-        switch ($type) {
-            case MAIL_TYPE_CERTIFICATE:
-                $domain = $payload['domain'];
-                $error = $payload['error'];
-                $attempt = $payload['attempt'];
-
-                $subject = \sprintf($locale->getText("$prefix.subject"), $domain);
-                $body->setParam('{{domain}}', $domain);
-                $body->setParam('{{error}}', $error);
-                $body->setParam('{{attempt}}', $attempt);
-                break;
-            case MAIL_TYPE_INVITATION:
-                $subject = \sprintf($locale->getText("$prefix.subject"), $team->getAttribute('name'), $projectName);
-                $body->setParam('{{owner}}', $user->getAttribute('name'));
-                $body->setParam('{{team}}', $team->getAttribute('name'));
-                break;
-            case MAIL_TYPE_RECOVERY:
-            case MAIL_TYPE_VERIFICATION:
-            case MAIL_TYPE_MAGIC_SESSION:
-                $subject = $locale->getText("$prefix.subject");
-                break;
-            default:
-                throw new Exception('Undefined Mail Type : ' . $type, 500);
-        }
```

Comment:
Do you know why all this was removed? Can you point me to the PR ?

Reply:
@christyjacob4 here you go
https://github.com/appwrite/appwrite/pull/4859